### PR TITLE
Added setRgbw function

### PR DESCRIPTION
**Pull request by mxtra**
Some MagicHome-Devices support white LEDs as well. Therefore, we should allow to change all channels with one command.

### CODE REVIEW

File: flux_led/__init__.py [hunk 1]
```diff
@@ -622,9 +635,9 @@ def setRgb(self, r,g,b, persist=True):
             msg = bytearray([0x31])
         else:
             msg = bytearray([0x41])
-        msg.append(r)
-        msg.append(g)
-        msg.append(b)
+        msg.append(int(r))
```
Comment (tchellomello) [inline flux_led/__init__.py]: I see that you created a new method setRgbw(). Why not implement a new one getRgbw(). 

Did you test if this does not break the current getRgb() on nonwhite lights?

Comment (mxtra) [inline flux_led/__init__.py]: I added getRgbw(). In  my setup, both functions work fine with both RGB and RGBW lights. Setting the white components does nothing and when getting it 0 is returned.

**Comment by Danielhiversen** [comment]
Looks fine.
@mxtra  Do want me to make a new release?

**Comment by mxtra** [comment]
Maybe we should wait and I'll add better brightness handling before you do that.

**Pull request merged**